articles/basics-of-transfer-learning/index.md

```diff
@@ -0,0 +1,148 @@
+
+#### Transfer Learning Approaches
+The strategies discussed above help us understand where transfer learning can be implemented. The approaches explored below shed some light on what can be transferred in the context of the above strategies.
+
+##### Instance Transfer
+In an ideal context, reusing all the knowledge from a source domain on a target task would be a common occurrence. Realistically, it is not possible to re-use data from the source domain directly. But, various instances from the source domain can be re-used. These instances of the source are transferred to the target task. These instances combined with data of the target domain define the instance transfer technique.
+
+##### Feature-Representation Transfer
+A feature may be defined as a property shared by independent units of data on which analysis should be done. The goal of feature representation transfer is to find suitable feature representations. These feature representations reduce the difference between the source and target domains. We are basically "transferring features" from the source domain to the target. This technique also reduces the error of classification and regression models. Supervised or unsupervised methods may be used for this technique but it is dependent on the availability of labeled data.
+
+##### Parameter Transfer
+Parameter transfer is based on two assumptions. The first one is that models with related tasks share some parameters. The second is that these models share prior distribution of hyperparameters. This technique, therefore, aims to discover shared parameters between source domain models and target domain models. Shared parameters make it possible to carry out transfer learning.
+
+##### Relational-Knowledge Transfer
+Relational-knowledge transfer maps out relational knowledge between source and target domains. This method involves data that is not independent and identically distributed. For example, networked data and social network data. The technique looks to transfer the relationship within data from a source to a target domain.
+
+For a deep dive into these transfer learning approaches, take a look at this [paper]( https://www.cse.ust.hk/~qyang/Docs/2009/tkde_transfer_learning.pdf).
+
+### Applications of Transfer Learning
+Several transfer learning algorithms are used in other related fields. Such as medicine, image detection, speech recognition, and recommendation systems to name a few. We will go over these in more detail in the following sections.
+
+#### Medicine
+Many algorithms have been proposed and implemented in medical imaging to reduce the workload of doctors. They also serve the purpose of improving the accuracy of judgment. One method involves using a [convolutional neural network (CNN)](https://en.wikipedia.org/wiki/Convolutional_neural_network) for model transfer. The goal is the segmentation of white matter lesions of multiple sclerosis (MS) in patient images.
```

Comment:
The problem statement needs some attention. Consider covid-19 detection using X-rays as an example or any other example that you find suitable. Restructure the problem statement to enhance readability.